### PR TITLE
docs: [upload] improve typings & fix style

### PR DESCRIPTION
**Pull request by sxzz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/.vitepress/theme/index.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import ElementPlus from 'element-plus'
 
-import 'uno.css'
-
 import VPApp, { globals, NotFound } from '../vitepress'
 import { define } from '../utils/types'
+import 'uno.css'
```
Comment (JeremyWuuuuu) [inline docs/.vitepress/theme/index.ts:5]: **Suggestion:** can we not include unrelated code?

**Review by jw-foss — COMMENTED** [review]
Left 1 comment for you. ;>

File: docs/.vitepress/theme/index.ts (hunk 1 above)
Comment (sxzz) [reply docs/.vitepress/theme/index.ts:5]: No, we should fix style

**Review by jw-foss — APPROVED** [review]
